nearpy/tests/experiments_tests.py

```diff
@@ -199,5 +199,69 @@ def test_random_binary_projections(self):
         print('\nRecall RBP: %f, Precision RBP: %f, SearchTime RBP: %f\n' % \
             (recall1, precision1, searchtime1))
 
+    def test_delete_vector(self):
+        # test case for 1 hash
+        dim = 5
+        rbp = RandomBinaryProjections('rbp', 10)
```

Comment:
Try [UniBucket hash](https://github.com/pixelogik/NearPy/blob/master/nearpy/hashes/unibucket.py#L28), it's less random.